packages/patoline/patoline.0.2/opam

```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Patoline typesetting system and libraries"
+description: """
+Patoline is a collection of typesetting libraries designed to be combined
+   into a variety of possible applications. Its primary goal is not to offer
+   the definitive answer to all typesetting problems, but rather to write the
+   painful and boring parts of the job, so that interesting tools could be
+   written easily. Obviously, a related (but much smaller) project is the
+   Patoline compiler, which compiles a mixed Wiki/LaTeX/Ocaml syntax into a
+   variety of output formats, including traditional PDF files, but also web
+   servers that deliver dynamic contents."""
+maintainer: ["Rodolphe Lepigre <lepigre@mpi-sws.org>"]
+authors: [
+  "Pierre-Étienne Meunier"
+  "Christophe Raffalli <christophe@raffalli.eu>"
+  "Rodolphe Lepigre <lepigre@mpi-sws.org>"
+  "Tom Hirschowitz"
+  "Florian Hatat <patoline@hatat.me>"
+]
+license: "GPL-2.0"
```

Comment:
I cannot fix that yet [here](https://github.com/patoline/patoline/blob/75fd8c928efc68d0aaa400d3a699a0e668c26c5f/dune-project#L15) because this requires dune 3.2, but I added a TODO note.